terra2/assetlist.json

```diff
@@ -510,4 +510,52 @@
         }
       }
     ]
+  },
+{
+  "denom_units": [
+    {
+      "denom": "ibc/A356EC90DC3AE43D485514DA7260EDC7ABB5CFAA0654CE2524C739392975AD3C",
+      "exponent": 0,
+      "aliases": [
+        "wstETH"
+      ]
+    },
+    {
+      "denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
+      "exponent": 6
+    }
+  ],
+  "type_asset": "ics20",
+  "base": "ibc/A356EC90DC3AE43D485514DA7260EDC7ABB5CFAA0654CE2524C739392975AD3C",
+  "name": "Canonical Wrapped stETH",
+  "display": "wstETH",
+  "symbol": "wstETH",
+  "traces": [
+    {
+      "type": "ibc",
+      "counterparty": {
+        "chain_name": "neutron",
+        "base_denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
```

Comment:
We need this asset to be registered to Neutron as well

Reply:
Another PR has just registered this asset to Neutron, so now we can reference it in traces here